graphene/src/auto/point.rs

```diff
@@ -31,13 +31,6 @@ impl Point {
         }
     }
 
-    #[cfg(feature = "v1_12")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "v1_12")))]
-    #[doc(alias = "graphene_point_distance_squared")]
-    pub fn distance_squared(&self, b: &Point) -> f32 {
```

Comment:
Is this correct?

Reply:
The nightly sdk downgraded graphene, only @alatiera can answer this

Reply:
That makes sense, thanks :)